packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -14,20 +14,31 @@ import { ExtensionSlot, age } from '@openmrs/esm-framework';
 interface PatientBannerProps {
   patient: fhir.Patient;
   patientUuid: string;
+  patientUrl?: string;
+  onClick?: (patientUrl: string) => void;
 }
 
-const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid }) => {
+const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, patientUrl, onClick }) => {
```

Comment:
Just remove `patientUrl` from the component altogether. Bake it into the `onClick` handler.

```suggestion
const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, onClick }) => {
```